src/store/data-explorer/data-explorer-reducer.ts

```diff
@@ -65,6 +65,9 @@ export const dataExplorerReducer = (state: DataExplorerState = {}, action: DataE
 export const getDataExplorer = (state: DataExplorerState, id: string) =>
     state[id] || initialDataExplorer;
 
+export const getSortColumns = (dataExplorer: DataExplorer) => dataExplorer.columns.find((c: any) =>
```

Comment:
I'd say it returns a single `column` not `columns` :)

Reply:
oops, typo :)